app/frontend/components/pages/accounts/accounts.tsx

```diff
@@ -0,0 +1,286 @@
+import {Fragment, h} from 'preact'
+import {connect} from '../../../helpers/connect'
+import actions from '../../../actions'
+import range from '../../../../frontend/wallet/helpers/range'
+import printAda from '../../../helpers/printAda'
+import {Lovelace} from '../../../state'
+import {AdaIcon} from '../../common/svg'
+import Alert from '../../common/alert'
+import SendTransactionModal from './sendTransactionModal'
+import DelegationModal from './delegationModal'
+import tooltip from '../../common/tooltip'
+
+const Account = ({
+  i,
+  account,
+  firstAddressPerAccount,
+  setAccount,
+  selectedAccount,
+  showDelegationModal,
+  showSendTransactionModal,
+}) => {
+  const isSelected = selectedAccount === i
```

Comment:
I would rather name the variable `selectedAccountIndex` so it's clear it's not an Account object

Reply:
You are right :+1: , fixed.